WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -432,6 +425,23 @@ public MySiteFragment getMySiteFragment() {
         return null;
     }
 
+    // Updates `last_seen` notifications flag in Simperium and removes tab indicator
+    private class UpdateLastSeenTask extends AsyncTask<Void, Void, Void> {
+        @Override
+        protected Void doInBackground(Void... voids) {
+            SimperiumUtils.updateLastSeenTime();
```

Comment:
I'm wondering if `updateLastSeenTime` should be a boolean that only returns `true` when `META_LAST_SEEN` has been updated, and then this AsyncTask should be a Boolean that enables us to skip updating the badge when `updateLastSeenTime` returns false. How likely is it that updating the last seen time wouldn't change anything?